### PR TITLE
Remove timeout in async_test for html tests

### PR DESCRIPTION
**Pull request by qiuzhong**
Remove all the timeout in `async_test` for html tests.
Affected tests: 125
Before: Pass: 64, Failed: 61
After:  Pass: 64, Failed: 61
Related: #11120

### CODE REVIEW

**Review by jdm — APPROVED** [review]

**Comment by jdm** [comment]
I saw those "no output" errors on several of the PRs removing the test-specific timeout values.

**Comment by jgraham** [comment]
It happens more now because we filtered out a lot of the process output from Firefox in order to avoid overflowing the logs, but the flip side is we experience more timeouts. If we wanted to fix it we'd presumably have to arrange for a periodic ping to be logged.

**Comment by foolip** [comment]
With https://github.com/web-platform-tests/wpt/pull/14096 merged it should be possible to just rebase this or otherwise retrigger checks.

**Comment by qiuzhong** [comment]
Found lots of errors like this:

```
Traceback (most recent call last):
  File "/home/test/web-platform-tests/tools/wptserve/wptserve/handlers.py", line 310, in __call__
    rv = self.func(request, response)
  File "/home/test/web-platform-tests/html/semantics/embedded-content/media-elements/track/track-element/cors/support/cors-tester.py", line 36, in main
    line = 'cors = ' + cors + ' | cookie = ' + cookie.value;
AttributeError: 'str' object has no attribute 'value'
```

File: html/browsers/browsing-the-web/history-traversal/browsing_context_name_cross_origin_2.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
 <pre id="step_log"></pre>
 <iframe id="test"></iframe>
 <script>
-var t = async_test(undefined, {timeout:10000});
+var t = async_test(undefined);
```
Comment (foolip) [inline html/browsers/browsing-the-web/history-traversal/browsing_context_name_cross_origin_2.html]: Can you also remove the undefined arguments? They were only there to make it possible to pass in the second argument.

**Review by foolip — APPROVED** [review]

**Comment by foolip** [comment]
Thanks @qiuzhong, one down, a few more PRs to go :)

**Comment by foolip** [comment]
https://tools.taskcluster.net/groups/DWgMz-OFQOKp5iktSe1gSg unfortunately failed so a clean before/after diff view for this PR isn't possible.

**Comment by foolip** [comment]
Filed https://github.com/web-platform-tests/wpt/issues/14207 for the fact that Taskcluster is failing. Once there's another working run I'll compare, but then more changes will be lumped in.

**Comment by foolip** [comment]
That's been resolved, here are diff views that includes the changes in this PR and more:
https://wpt.fyi/results/html?product=chrome%4039e6009b92&product=chrome%405438f72ea0&diff
https://wpt.fyi/results/html?product=firefox%4039e6009b92&product=firefox%405438f72ea0&diff

Just one seeming regression:
https://wpt.fyi/results/html/semantics/embedded-content/the-iframe-element/iframe_sandbox_popups_escaping-1.html?product=chrome%4039e6009b92&product=chrome%405438f72ea0&diff

Not quite regressions, see https://github.com/web-platform-tests/wpt.fyi/issues/796:
https://wpt.fyi/results/html/semantics/scripting-1/the-script-element/module/dynamic-import/no-active-script-manual-classic.html?product=chrome%4039e6009b92&product=chrome%405438f72ea0&diff
https://wpt.fyi/results/html/semantics/scripting-1/the-script-element/module/dynamic-import/no-active-script-manual-module.html?product=chrome%4039e6009b92&product=chrome%405438f72ea0&diff
https://wpt.fyi/results/html/semantics/scripting-1/the-script-element/module/dynamic-import/no-active-script-manual-classic.html?product=firefox%4039e6009b92&product=firefox%405438f72ea0&diff
https://wpt.fyi/results/html/semantics/scripting-1/the-script-element/module/dynamic-import/no-active-script-manual-module.html?product=firefox%4039e6009b92&product=firefox%405438f72ea0&diff

**Comment by foolip** [comment]
Ah, but iframe_sandbox_popups_escaping-1.html wasn't modified, so that's probably a flaky test.

**Pull request merged**